driver/debug/debug.go

```diff
@@ -0,0 +1,47 @@
+package debug
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/cnabio/cnab-go/driver"
+)
+
+// Driver prints the information passed to a driver
+//
+// It does not ever run the image.
+type Driver struct {
+	config map[string]string
+}
+
+// Run executes the operation on the Debug driver
+func (d *Driver) Run(op *driver.Operation) (driver.OperationResult, error) {
```

Comment:
Can we pass a context object as well to handle things like deadlines/cancellation/tracing?

```suggestion
func (d *Driver) Run(ctx context.Context, op *driver.Operation) (driver.OperationResult, error) {
```

Reply:
I started making this change here, but I think it would be better in a second PR so we can really look over the changes to implementing the context handling in the drivers in isolation. I'll merge this then immediately open up the PR with my changes and tests so far and we can see what else needs to be done.